python/requirements_build_wheel.txt

```diff
@@ -0,0 +1 @@
+virtualenv
```

Comment:
If there is only one requirement, do we need a file?
Especially, that this is not really required to build a wheel. It's only required **if** you're using the make target.
I would recommend not adding this file.